### ISSUE TITLE
Sandbox: Aktywacja konta firmowego 

### ISSUE THREAD

**Issue by PLoO95** [allegro/allegro-api#6740]
Dzień dobry

Proszę o aktywacje konta firmowego:
Login: hpysz

**Comment by PrzemyslawLukanowski** [comment]
Uzupełnij proszę wszystkie dane zgodnie z krokowcem na stronie. Na koncie brakuje m.in. numeru NIP i konta bankowego. Na poniższym etapie wybierz weryfikację pełną i prześlij dowolne dokumenty.
![Screenshot 2023-02-10 at 09 43 17](https://user-images.githubusercontent.com/51473661/219378712-27b4bb6e-9bb8-4c9c-9553-1a48f95a8de0.png)

**Comment by PLoO95** [comment]
Dodałem wygenerowany numer konta (fekowy IBAN)
Oraz uzupełniłem dane weryfikacji konta (fakowymi danymi)

![image](https://user-images.githubusercontent.com/16743412/219380241-732e3dba-be85-4ae7-a70d-be927fa9988c.png)

**Comment by PrzemyslawLukanowski** [comment]
Dziękuję, aktywowałem Twoje konto.

**Comment by MatMark** [comment]
Dzień dobry,

można również prosić o weryfikację konta firmowego?
Login: MatMarkHex

**Comment by Lukasz-Zurek** [comment]
@MatMark Twoje konto posiada status zwykłego. Możesz zmienić rodzaj konta w tej [zakładce](https://allegro.pl.allegrosandbox.pl/moje-allegro/moje-konto/dane-konta) (opcja "zmień na firmowe"). Następnie przejdź przez cały "krokowiec". W razie jakichkolwiek problemów - daj proszę znać.

**Comment by MatMark** [comment]
Już raz tam wchodziłem i wypełniałem dane, teraz jak tam chodzę widzę taki ekran, myślałem że status zwykłego konta jest ze względu na to, że dane firmowe nie zostały jeszcze zweryfikowane
![obraz](https://user-images.githubusercontent.com/37310556/220043861-8b89074f-1bc8-4f53-a0d0-ea4beb84df64.png)

**Comment by Lukasz-Zurek** [comment]
@MatMark Nie, w tym przypadku konto jest zwykłe i musisz przekonwertować je na firmowe. Przejdź na [tę](https://allegro.pl.allegrosandbox.pl/moje-allegro/zakupy/konwersja-konta) stronę, następnie po wyborze rodzaju działalności i wypełnieniu danych zostaniesz przekierowany na [stronę wyboru ścieżki](https://allegro.pl.allegrosandbox.pl/rejestracja/konto-firmowe/wybierz-sciezke) - wybierz tę dotyczącą sprzedaży. Na końcu będziesz musiał [ustawić wypłaty](https://allegro.pl.allegrosandbox.pl/rejestracja/konto-firmowe/ustaw-wyplaty) i ponownie uzupełnić formularz danych i załączyć dokumenty (fikcyjne) firmy oraz wykonać przelew online (za pośrednictwem naszego symulatora płatności).

**Comment by MatMark** [comment]
@Lukasz-Zurek Hmmm to musiało się coś pochrzanić z moim kontem chyba, bo te wszystkie wspomniane kroki już robiłem, a gdy próbuję wejść w ten [link](https://allegro.pl.allegrosandbox.pl/moje-allegro/zakupy/konwersja-konta) od razu przekierowuje mnie do ustawień wpłat (wcześniejszy zrzut ekranu)
![obraz](https://user-images.githubusercontent.com/37310556/220053105-a3cc8d6a-cdfc-4fcd-9eeb-fcc6d2c5255d.png)
a link do [wyboru ścieżki](https://allegro.pl.allegrosandbox.pl/rejestracja/konto-firmowe/wybierz-sciezke) po chwili przekierowuje na błąd 404
![obraz](https://user-images.githubusercontent.com/37310556/220053712-e6cd4d62-8a0f-481f-8e40-e21bbc93c1f4.png)
Przy płatnościach mam informację o tym, że dane firmy są weryfikowane (przelew również ponawiałem)
![obraz](https://user-images.githubusercontent.com/37310556/220054125-02a52324-bfa8-4010-864d-d5723e3b0045.png)

**Comment by Lukasz-Zurek** [comment]
@MatMark Ok, w tej sytuacji przepraszam za te utrudnienia. Muszę niestety zasugerować Tobie rejestrację nowego konta. Podczas rejestracji wybierz konto firmowe. Oczywiście w razie problemów z końcowym etapem weryfikacji, chętnie Tobie pomogę.

**Comment by MatMark** [comment]
Ok, założyłem nowe konto, teraz wygląda to trochę inaczej, tym razem już chyba brakuje tylko weryfikacji (Login: MatMark)
![obraz](https://user-images.githubusercontent.com/37310556/220065074-f62f1888-89f2-4ace-8bec-4da561bd0b55.png)

**Comment by Lukasz-Zurek** [comment]
@MatMark Zweryfikowałem już konto :) Sprawdź proszę czy wszystko jest ok.

**Comment by MatMark** [comment]
Wydaje się ok, chociaż przy weryfikacji danych firmy dalej nie ma potwierdzenia, ale przedmioty mogę wystawiać itp. 
![obraz](https://user-images.githubusercontent.com/37310556/220074831-b0d165fd-2996-4cc1-a1e0-ab7924f7e2d5.png)

**Comment by Lukasz-Zurek** [comment]
Do usług :) Odświeżyłem jeszcze krokowiec, powinno być ok (podkreślam "powinno", bo to Sandbox ;)). Niemniej jednak nie wpływa to na wystawianie, etc.

**Comment by bennymerkle** [comment]
Can you verify our account for API?
Login: sell-store

**Comment by Lukasz-Zurek** [comment]
@bennymerkle I cannot find such a login in our test environment (Sandbox) database.

**Comment by bennymerkle** [comment]
@Lukasz-Zurek 
We have an Allegro Business Account and want to use your Shopware 6 connector.
The login to "allegro.pl" works. Do I have to create a second account for the sandbox?

**Comment by Lukasz-Zurek** [comment]
@bennymerkle Yes, you will have to create a separate business account on the sandbox. But please note that it will be more simplified process because you will not be obligated to make a real payment - you can use our payment simulator where you will have to accept it and fill all of the forms but we recommend using fictitious data. If there is any problem with a verification process - please just let me know :)

**Comment by bennymerkle** [comment]
@Lukasz-Zurek We want to use your Shopware 6 Plugin to import real sales. The plugin wants to connect to the sandbox instead of allegro.pl.

**Comment by Lukasz-Zurek** [comment]
@bennymerkle Please contact directly Shopware. We are not responsible for this plugin so I am not able to help you. I thought that you would like to test our resources freely so I suggested creating an account on the Sandbox.

**Comment by Feynman00** [comment]
Dzień dobry, 
Proszę o weryfikacje konta Klocuch123
![2023-02-28_09h29_11](https://user-images.githubusercontent.com/124243384/221797256-558856b2-fe8e-4b3c-bea2-06cd6c233568.png)

**Comment by Feynman00** [comment]
Proszę o aktywacje konta Klocuch123
![2023-02-28_09h29_11](https://user-images.githubusercontent.com/124243384/221878844-1eb7edc4-29bd-4b55-b54e-fa531e80ecd3.png)

**Comment by MartaNowaczyk** [comment]
@Feynman00 konto aktywowane. Zobacz czy teraz jest ok.

**Comment by Lukasz-Zurek** [comment]
@Feynman00 Konto jest zweryfikowane. Proszę spróbować ponownie.

**Comment by Feynman00** [comment]
działa, dziękuje :)

**Comment by mieszko-dev** [comment]
Dzień dobry,

Proszę o aktywację konta firmowego: mmmmmmmmm

**Comment by MartaNowaczyk** [comment]
@mieszko-dev konto powinno być już aktywne.

**Comment by CSVK2022** [comment]
Bardzo proszę o aktywację konta CSVK_SB

**Comment by Lukasz-Zurek** [comment]
@CSVK2022 Aktywowałem i zweryfikowałem Twoje konto, o czym pisałem już [tutaj](https://github.com/allegro/allegro-api/issues/1046#issuecomment-1459622683).

**Comment by Kacper3k** [comment]
Prosiłbym o aktywacje konta artmobi

**Comment by AureliuszBrussy** [comment]
@Kacper3k aktywowałem Twoje konto.
